layout-server/src/test/java/org/finos/vuu/layoutserver/model/LayoutTest.java

```diff
@@ -0,0 +1,27 @@
+package org.finos.vuu.layoutserver.model;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.verify;
+
+@ExtendWith(MockitoExtension.class)
+class LayoutTest {
+
+        @Test
+        void setId_anyId_setsIdForBothLayoutAndMetadata() {
+            UUID id = UUID.fromString("00000000-0000-0000-0000-000000000000");
+            Layout layout = new Layout();
+            Metadata metadata = new Metadata();
+
+            layout.setMetadata(metadata);
+            layout.setId(id);
+
+            assertThat(layout.getId()).isEqualTo(id);
```

Comment:
Really nit-picky, but I often like to format so that the action under test (`setId` in this case) is clearly isolated:

```suggestion
            Metadata metadata = new Metadata();
            layout.setMetadata(metadata);

            layout.setId(id);

            assertThat(layout.getId()).isEqualTo(id);
```